internal/kibana/url_prefixes.go

```diff
@@ -24,6 +26,18 @@ const (
 
 // getBaseURL will pull in the baseurl or an alternative host based on settings
 func getBaseURL() string {
+	// If a custom KIBANA URL is passed, use that instead
+	remoteKibanaHost := shell.GetEnv("KIBANA_URL", "")
```

Comment:
Can we add this variables to the README doc where we explain the variables affecting the build? https://github.com/elastic/e2e-testing/tree/master/e2e#environment-variables-affecting-the-build